pkg/orgIdColumnPopulator/main.go

```diff
@@ -0,0 +1,161 @@
+package orgIdColumnPopulator
```

Comment:
\<nitpick\/\> gophers won't like this package name - https://go.dev/blog/package-names

Reply:
Yeah, I'm not sure what to call this.  I think I'll go with tenantconv which is a similar to the standard strconv package.  Maybe the gophers will let me pass??   :)